README.md

```diff
@@ -1,250 +1,326 @@
-creation and apply patches / integrates custom freifunk packages. All custom
-patches are located in *patches/* and all additional packages can be found at
-http://github.com/freifunk-berlin/packages_berlin.
+How does this stuff work? 
+
+The idea is to download OpenWrt via git, patch it with all patches in the
+`patches/` folder and configure it according to the configuration snippets in
+`configs/`.  Then use the OpenWrt buildsystem to build all packages and the
+Imagebuilder and use the latter to assemble the final image with lists of
+packages in packages for the router. All berlin-specific additional packages 
+can be found at https://github.com/freifunk-berlin/firmware-packages.
+
 
 ### Build Prerequisites
 
-Please take a look at the [OpenWrt documentation](https://openwrt.org/docs/guide-developer/build-system/install-buildsystem?s[]=prerequisites#prerequisites)
+Please take a look at the [OpenWrt
+documentation](https://openwrt.org/docs/guide-developer/build-system/install-buildsystem?s[]=prerequisites#prerequisites)
 for a complete and uptodate list of packages for your operating system. Make
 sure the list contains `quilt`. We use it for patch management.
 
-On Ubuntu/Debian:
+On Ubuntu/Debian: 
 ```
 apt-get install git build-essential libncurses5-dev zlib1g-dev gawk time \
-  unzip libxml-perl flex wget gawk gettext quilt python libssl-dev
+unzip libxml-perl flex wget gawk gettext quilt python libssl-dev 
 ```
 
-On openSUSE:
+On openSUSE: 
+``` 
+zypper install --type pattern devel_basis zypper install git ncurses-devel \
+zlib-devel gawk time unzip perl-libxml-perl flex wget gawk gettext-runtime \
+quilt python libopenssl-devel 
 ```
-zypper install --type pattern devel_basis
-zypper install git ncurses-devel zlib-devel gawk time \
-  unzip perl-libxml-perl flex wget gawk gettext-runtime quilt python libopenssl-devel
-```
-On Arch/Antergos:
+On Arch/Antergos: 
 ```
 pacman -S base-devel git ncurses lib32-zlib gawk time unzip perl-xml-libxml \
- flex wget gettext quilt python2 openssl
+flex wget gettext quilt python2 openssl 
 ```
 
-### Building all firmwares
 
-To get the source and build the firmware locally use:
+### Build-System-Structure  
+
+Where can I change something? What do these files do? 
+
+``` 
+:: Settings
+
+config.mk         - select what package lists to build
+configs/          - 
+modules           - define git repository, revision, etc of OpenWRT and Packages
+
+patches/          - Patches against the OpenWRT directory  
+packagelists/     - Package Lists  
+profiles/         - List of router profiles for each architecture - names come 
+                      from the Imagebuilder
+embedded-files/   - Files that will be included in the images without further
+                      processing. Its mirrored to / . Thus, you must provide a
+                      directory tree there, to put your file at the right point
+
+:: Building 
+
+Makefile                - Does all the stuff. Cloning and Updating OpenWRT,
```

Comment:
* use updating in lower-case
* it's OpenWrt

Reply:
Done with regular expression. Should be fine now.